src/logic/operate.js

```diff
@@ -0,0 +1,27 @@
+import Big from 'big.js';
+
+const Operate = (numberOne, numberTwo, operation) => {
+  const nOne = Big(numberOne);
+  const nTwo = Big(numberTwo);
+  switch (operation) {
+    case '+':
+      return (nOne.plus(nTwo)).toString();
+    case '-':
+      return (nOne.minus(nTwo)).toString();
+    case 'x':
+      return (nOne.times(nTwo)).toString();
+    case '÷':
+      if (nOne !== 0 && nTwo !== 0) {
```

Comment:
0 / num is well known in mathematics as zero, the only thing you need to check is the other way around so num / 0 this is when we have a problem